### PR TITLE
TVR-8626

### PR DESCRIPTION
**Pull request by laurentlhermine**
New documentation from TVR-8626
Remove back to top links
Precision about the userId parameter
Get list of requests endpoint and example
New response codes

### CODE REVIEW

File: src/api-reference/request/v4.request.markdown [hunk 1]
```diff
@@ -1,4 +1,4 @@
----
+# Ne---
```
Comment (charlieconcur) [inline src/api-reference/request/v4.request.markdown]: What is this...?

File: src/api-reference/request/v4.examples.markdown [hunk 2]
```diff
@@ -148,20 +149,103 @@ Responde Body:
...
     }
   ]
 }
 ```
 [Back to top](#request-examples)
 
+#### <a name="request-resource-get-list"></a>Get the list of existing Requests
+#### GET /v4/requests
+```
+curl -X GET https://us.api.concursolutions.com/travelrequest/v4/requests?view=ALL&limit=10&start=0 --header 'Accept: application/json' --header 'Authorization: Bearer {token}'
```
Comment (charlieconcur) [inline src/api-reference/request/v4.examples.markdown]: We need to remove references to specific tools. Follow the example given in the [Documentation Template](https://github.com/concur/developer.concur.com/blob/preview/src/document-template.markdown).

File: src/api-reference/request/v4.request.markdown (hunk 1 above)
Comment (cbobrie) [reply src/api-reference/request/v4.request.markdown]: A bad copy paste from the markdown editor page => --- must be kept

File: src/api-reference/request/v4.examples.markdown (hunk 2 above)
Comment (laurentlhermine) [reply src/api-reference/request/v4.examples.markdown]: Is this really required ? curl is a pretty standard cross platform tool. I think it really helps developers to figure out what a valid call is.

File: src/api-reference/request/v4.request.markdown (hunk 1 above)
Comment (laurentlhermine) [reply src/api-reference/request/v4.request.markdown]: corrected

File: src/api-reference/request/v4.examples.markdown (hunk 2 above)
Comment (cbobrie) [reply src/api-reference/request/v4.examples.markdown]: Hi Charlie. There seems to be some current discussion wth Carol on this subject, however as we are externalizing the Get list of Request this saturday, and knowing that removing the curls will require dev time not planned and possible before saturday - of confirmed, please display the documentation with Curl, and for the october release (there will be new documentation update) we'll see what is really necessary and what can be done on the dev side to make it possible.
Thanks in advance.

**Pull request merged**